sdcm/cluster.py

```diff
@@ -4227,7 +4227,17 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
             if install_scylla:
                 self._scylla_install(node)
             else:
+                self.log.info("Waiting for preinstalled Scylla")
                 self._wait_for_preinstalled_scylla(node)
+                self.log.info("Done waiting for preinstalled Scylla")
+
+                if self.params.get('workaround_kernel_bug_for_iotune'):
+                    self.log.info("This AMI need to be tweaked for io.conf and properties")
+                    for conf in ['io.conf', 'io_properties.yaml']:
```

Comment:
The io.conf and io_properties.yaml in side configurations/ are prepared for default GCE instance ('n1-highmem-8' (8vcpus/52Gmem)).

Do they match with m5d.large (2vcpus / 8G mem)?

Reply:
It doesn’t really matter for the test. We use 200 nodes, so the stress per node is very very small.

Reply:
Then I'm ok with this.